test/lib/ufe/testAttributeBlock.py

```diff
@@ -347,6 +349,83 @@ def testAttributeBlocking3dMatrixOps(self):
 
         # authoring new transformation edit is allowed.
         self.assertTrue(mayaUsdUfe.isAttributeEditAllowed(transformAttr))
+    
+    def testIsAttributeEditAllowedFunction(self):
+        rootLayerStr = """\
```

Comment:
Please add a function doc explaining the goal of the test.

Reply:
Will do, thanks.